### PR TITLE
Add Reload task

### PR DESCRIPTION
**Pull request by alexndlm**
Add Reload task and small fixes

### CODE REVIEW

File: src/Reload.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * This file is part of RoboSystemService.
+ *
+ * @author      Aitor García Martínez (Falc) <aitor.falc@gmail.com>
+ * @copyright   2015 Aitor García Martínez (Falc) <aitor.falc@gmail.com>
```
Comment (Falc) [inline src/Reload.php]: Remove my `@author` and `@copyright` tags from this file, I didn't write anything in it.

File: tests/ReloadTest.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * This file is part of RoboSystemService.
+ *
+ * @author      Aitor García Martínez (Falc) <aitor.falc@gmail.com>
+ * @copyright   2015 Aitor García Martínez (Falc) <aitor.falc@gmail.com>
```
Comment (Falc) [inline tests/ReloadTest.php]: Remove my `@author` and `@copyright` tags from this file, I didn't write anything in it.

Comment (alexndlm) [reply tests/ReloadTest.php]: Done.

File: src/Reload.php (hunk 1 above)
Comment (alexndlm) [reply src/Reload.php]: Done

**Comment by Falc** [comment]
Thank you, @alexndlm :)

**Pull request merged**